android/cli/commands/_build.js

```diff
@@ -3380,8 +3380,16 @@ AndroidBuilder.prototype.generateTheme = function generateTheme(next) {
 			flags += '.Fullscreen';
 		}
 
+		let theme = flags;
+		if (this.tiappAndroidManifest && this.tiappAndroidManifest.application && this.tiappAndroidManifest.application.theme) {
+			if (theme.startsWith('@style/') && theme !== '@style/Theme.Titanium.Translucent') {
```

Comment:
Maybe `theme` should be `this.tiappAndroidManifest.application.theme` in the condition?

Reply:
`theme` gets set to that only when a custom theme has been defined

https://github.com/appcelerator/titanium_mobile/pull/9581/files#diff-c5b02bc76af40eccf082f63f6da1ec91R3386